apps/common/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  name: 'common',
+  steps: {
+    '/start': {
+      next: 'ima/continue-form'
+    },
```

Comment:
Sorry missed this one, can we do without the start step or is it needed to build?

Reply:
Removed